packages/eas-cli/src/commandUtils/context/contextUtils/getProjectIdAsync.ts

```diff
@@ -93,9 +93,22 @@ export async function getProjectIdAsync(
     const appForProjectId = await AppQuery.byIdAsync(graphqlClient, localProjectId);
     if (exp.owner && exp.owner !== appForProjectId.ownerAccount.name) {
       throw new Error(
-        `Project config: Project identified by "extra.eas.projectId" (${
+        `Project config: Owner of project identified by "extra.eas.projectId" (${
           appForProjectId.ownerAccount.name
-        }) is not owned by owner specified in the "owner" field (${exp.owner}). ${learnMore(
+        }) does not match owner specified in the "owner" field (${exp.owner}). ${learnMore(
+          'https://expo.fyi/eas-project-id'
+        )}`
+      );
+    }
+
+    const actorUsername = getActorUsername(actor);
+    if (!exp.owner && appForProjectId.ownerAccount.name !== actorUsername) {
+      throw new Error(
+        `Project config: Owner of project identified by "extra.eas.projectId" (${
+          appForProjectId.ownerAccount.name
+        }) does not match the logged in user (${actorUsername}) and the "owner" field is not specified. To ensure all libraries work correctly, "owner": "${
```

Comment:
If I understand correctly, it's a user-facing error message. "libraries" in this context probably don't mean anything to our users. I'd suggest avoiding using this term.

Reply:
We use the term in developer documentation when referring to libraries (which are what is broken by not having an owner field) so I think I lean towards keeping it: https://docs.expo.dev/push-notifications/push-notifications-setup/#install-libraries